debug/testlib.py

```diff
@@ -947,6 +948,33 @@ def __enter__(self):
     def __exit__(self, _type, _value, _traceback):
         self.gdb.pop_state()
 
+
+def load_excluded_tests(excluded_tests_file, target_name):
+    result = []
+    if excluded_tests_file is None or len(excluded_tests_file) == 0:
+        return result
+
+    target_excludes = {}
+    with open(excluded_tests_file) as file:
+        raw_data = yaml.safe_load(file)
+        invalid_format_message = "Invalid format for the exclusion file. Please take a look at the " \
+                                 "\"excluded.yaml.example\" file as an example of the test exclusion file format."
+        for test_list in raw_data.values():
+            if not isinstance(test_list, list):
+                raise ValueError(invalid_format_message, excluded_tests_file, test_list)
+            if not all(isinstance(s, str) for s in test_list):
+                raise ValueError(invalid_format_message, excluded_tests_file)
```

Comment:
And here, "not every element in the test list is a string"

Reply:
Added with the target name